app/components/pagination/PaginationSelectionFooter.tsx

```diff
@@ -9,40 +9,51 @@ import { useSubmit } from '@remix-run/react'
 import { Select } from '@trussworks/react-uswds'
 
 import Pagination from './Pagination'
+import { calculateLimit } from '~/lib/utils'
 
 export default function PaginationSelectionFooter({
   page,
   totalPages,
   limit,
   query,
   form,
+  view,
 }: {
   page: number
   totalPages: number
   limit?: number
   query?: string
   form: string
+  view?: string
 }) {
   const submit = useSubmit()
+  const isGroupView = view === 'group'
+  const limitValue = calculateLimit({ isGroupView, limit })
+
   return (
     <div className="display-flex flex-row flex-wrap">
       <div className="display-flex flex-align-self-center margin-right-2 width-auto">
         <div>
+          <input type="hidden" form={form} name="view" id="view" value={view} />
           <Select
             id="limit"
             title="Number of results per page"
             className="width-auto height-5 padding-y-0 margin-y-0"
             name="limit"
-            defaultValue="100"
+            value={limitValue}
             form={form}
             onChange={({ target: { form } }) => {
               submit(form)
             }}
           >
             <option value="10">10 / page</option>
             <option value="20">20 / page</option>
-            <option value="50">50 / page</option>
-            <option value="100">100 / page</option>
+            {!isGroupView && (
+              <>
+                <option value="50">50 / page</option>
+                <option value="100">100 / page</option>
+              </>
+            )}
```

Comment:
Why don't we show options for 50 or 100 results per page for the group view?

Reply:
each group could have many circulars. If each group had on average 10 circulars, if I included 100 groups at a time, that would be querying 1000 circulars to load the page. It seemed reasonable to limit it. Do you think large circular queries would be problematic?

Reply:
Not necessarily. I would prefer not limiting it for now.